src/app/subscriptions.ts

```diff
@@ -48,7 +60,19 @@ export function initializeSubscriptions(store: AppStore) {
   setRdt(rdtInstance);
   // TODO: "black" on the light theme
   rdtInstance.buttonApi.setTheme("white");
-  adex.init("stokenet");
+  let network;
+  switch (process.env.NETWORK) {
+    case "mainnet":
+      network = adex.ApiNetworkOptions.indexOf("mainnet");
+      break;
+    case "stokenet":
+      network = adex.ApiNetworkOptions.indexOf("stokenet");
+      break;
+    default:
+      network = adex.ApiNetworkOptions.indexOf("stokenet");
+  }
+
+  adex.init(adex.ApiNetworkOptions[network]);
```

Comment:
```suggestion
  adex.init(process.env.NETWORK || "stokenet");
```

Reply:
This was the first thing I tried, but ran into type issues.
The adex.init() is a bit strange it wants to have alphadexApiNetwork
![image](https://github.com/DeXter-on-Radix/website/assets/3978285/1f40a6c2-6fb8-44e3-95f0-8007b2a2b7c8)

![image](https://github.com/DeXter-on-Radix/website/assets/3978285/6b4b3ccc-177e-47ff-a462-3d0b9cbf61a3)